### PR TITLE
.fd-column visibility

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?235825

The `_this.$f.find('.fd-content-right .fd-column').addClass('hide');` lines fix the bug, other changes are minor cleanup.

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -1475,10 +1477,9 @@ define([
         this.$f.find('.fd-default-panel').addClass('hide');
 
         /* update display */
```
Comment (emord) [inline src/core.js]: nit: looks a little weird to have 

```
code
(whitespace)
comment
(whitespace)
code
```

**Comment by emord** [comment]
Thanks for updating the whole file to one way of doing things. What's the reason to prefer `$element.addClass('hide')` to `$element.hide()`?

Test would also be nice (and you could take credit for test 1000 :) )

File: src/core.js (hunk 1 above)
Comment (orangejenny) [inline src/core.js]: i suppose that comment wasn't adding value anyway

**Comment by orangejenny** [comment]
I do like round numbers. Test added.

> What's the reason to prefer $element.addClass('hide') to $element.hide()?

I tend to avoid `show` because it unilaterally sets `display` to `block`, which isn't always desirable (most often tripped up by this when hiding and then re-showing something that was previously displayed `inline-block`). At some point jQuery fixed this (possibly https://jquery.com/upgrade-guide/3.0/#breaking-change-show-hide-and-toggle-methods-now-respect-more-stylesheet-changes) but 1.11 uses the the bad behavior.

**Comment by emord** [comment]
Thanks for the explanation, makes a lot of sense (although maybe we won't need to worry in a couple of weeks :) )

:+1: on :white_check_mark:

**Pull request merged**